src/design-system/playground/Playground.tsx

```diff
@@ -0,0 +1,109 @@
+import React, { useCallback, useEffect, useState } from 'react';
+import {
+  ScrollView,
+  StatusBar,
+  StyleSheet,
+  TouchableOpacity,
+  View,
+} from 'react-native';
+import { useHideSplashScreen } from '../../hooks';
+import { Heading, Inline, Inset, Stack } from '../';
+import backgroundDocs from '../components/BackgroundProvider/BackgroundProvider.docs';
+import bleedDocs from '../components/Bleed/Bleed.docs';
+import boxDocs from '../components/Box/Box.docs';
+import columnsDocs from '../components/Columns/Columns.docs';
+import headingDocs from '../components/Heading/Heading.docs';
+import inlineDocs from '../components/Inline/Inline.docs';
+import insetDocs from '../components/Inset/Inset.docs';
+import markdownTextDocs from '../components/MarkdownText/MarkdownText.docs';
+import rowDocs from '../components/Row/Row.docs';
+import stackDocs from '../components/Stack/Stack.docs';
+import textDocs from '../components/Text/Text.docs';
+import textLinkDocs from '../components/TextLink/TextLink.docs';
+import { Docs } from './Docs';
+
+const allDocs = [
+  backgroundDocs,
+  bleedDocs,
+  boxDocs,
+  columnsDocs,
+  headingDocs,
+  inlineDocs,
+  insetDocs,
+  markdownTextDocs,
+  rowDocs,
+  stackDocs,
+  textDocs,
+  textLinkDocs,
+];
+
+const styles = StyleSheet.create({
+  docsRowToggle: {
+    flexDirection: 'row',
+    justifyContent: 'center',
+    width: 14,
+  },
+  layoutContainer: {
+    backgroundColor: 'rgba(0,0,0,0.05)',
+  },
+});
+
+const DocsRow = ({ name, category, examples }: Docs) => {
+  const [open, setOpen] = useState(false);
+
+  return (
+    <Stack space="30px">
+      <TouchableOpacity
+        onPress={useCallback(() => setOpen(x => !x), [setOpen])}
+      >
+        <Inline space="6px">
+          <View style={styles.docsRowToggle}>
+            <Heading size="20px">{open ? '-' : '+'}</Heading>
+          </View>
+          <Heading size="20px">{name}</Heading>
+        </Inline>
+      </TouchableOpacity>
+      {open
+        ? examples.map(({ name, Example }, index) => (
+            <Stack key={index} space="12px">
+              <Heading size="18px" weight="bold">
+                {name}
+              </Heading>
+              <View
+                style={
+                  category === 'Layout' && name !== 'Box'
+                    ? styles.layoutContainer
+                    : undefined
+                }
+              >
+                <Example />
+              </View>
+            </Stack>
+          ))
+        : null}
+    </Stack>
+  );
+};
+
+export const Playground = () => {
+  const hideSplashScreen = useHideSplashScreen();
+  useEffect(hideSplashScreen, [hideSplashScreen]);
+
+  return (
+    <ScrollView contentInsetAdjustmentBehavior="automatic">
+      {android ? <View style={{ height: StatusBar.currentHeight }} /> : null}
+      <Inset space="19px">
```

Comment:
I think I still advocate for the usage of numbers allowed (following our private chat). I see advantages of not doing this. Maybe worth discussing with a team?

Reply:
Last job, we created a design system where you could use `sm`, `md`, `lg` etc from our theme OR use a random number value for custom values. The problem with named values is when your system grows and you want to find some value in between `sm` and `md`, you have to name things in a crazy way, like `sm-L`. The problem of course with using _any_ number is that it's easy to build things that don't match a design/system perfectly.

I really like the tradeoff @markdalgleish showed me with headings, where you could specify a color value from our theme, and if you needed a [custom value](https://github.com/rainbow-me/rainbow/pull/2558/files#diff-4427e8b3a4d5015295b295ec265a3e97f49e4b1a5469a9780eaaf4fcbecf96a4R30), you used a prop like `color={{ custom: 'red' }}`. Wondering if that would be a good thing to include with certain components spacing?

Reply:
For now at least, I'm going with the format @nickbytes mentioned: `space={{ custom: 17 }}`. I think it strikes a good balance between encouraging the use of standard spacing values while still allowing you to explicitly step outside the system when needed. Adding this now.